part-spec/power.json

```diff
@@ -0,0 +1,202 @@
+                "examples": [
+                    "preProduction",
+                    "active",
+                    "EOL"
+                ],
+                "type": "string"
+            },
+            "regulatorTopology": {
+                "description": "switching voltage regulator topology",
+                "examples": [
+                    "buck",
+                    "boost",
+                    "buck-boost",
+                    "other"
+                ],
+                "type": "string"
+            },
+            "vinMin": {
+                "description": "minimum input voltage under which the part can be expected to operate properly",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "vinMax": {
+                "description": "maximum input voltage under which the part can be expected to operate properly",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "voutMin": {
+                "description": "minimum output voltage the part can regulate",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "voutMax": {
+                "description": "maximum output voltage the part can regulate",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "feedbackVoltage": {
+                "description": "voltage comparison point at the feedback node",
+                "comment": "units of volts",
+                "$ref": "#/definitions/unit"
+            },
+            "voutAccuracy": {
+                "description": "output voltage variation at no load",
+                "comment": "units of %",
+                "type": "number"
+            },
+            "loadRegulation": {
+                "description": "output voltage variation from no load to full load ",
+                "comment": "units of %",
+                "type": "number"
+            },
+            "lineRegulation": {
+                "description": "output voltage variation from minimum input voltage to maximum input voltage ",
+                "comment": "units of %",
+                "type": "number"
+            },
+            "Iq": {
+                "description": "quiescent current of voltage regulator at no load",
+                "comment": "units of amps",
+                "$ref": "#/definitions/unit"
+            },
+            "Isd": {
+                "description": "shutdown current of voltage regulato",
+                "comment": "units of amps",
+                "$ref": "#/definitions/unit"
+            },
+            "fsw": {
+                "description": "swtiching frequency of voltage regulator",
+                "comment": "units of hertz",
+                "$ref": "#/definitions/unit"
+            },
+            "ilimHSFET": {
```

Comment:
Consider pulling this into an object (perhaps in this power file) and then both regulator and power fets reference this